components/mpas-ocean/cime_config/buildnml

```diff
@@ -751,6 +751,29 @@ def buildnml(case, caseroot, compname):
         lines.append('    <var name="oceanHeatContent2000mToBot"/>')
         lines.append('</stream>')
         lines.append('')
+        lines.append('<stream name="mixedLayerHeatBudgetOutput"')
+        lines.append('        type="output"')
+        lines.append('        precision="single"')
+        lines.append('        io_type="{}"'.format(ocn_pio_typename))
+        lines.append('        filename_template="mpaso.hist.am.mixedLayerHeatBudget.$Y-$M-$D.nc"')
+        lines.append('        filename_interval="00-01-00_00:00:00"')
+        lines.append('        reference_time="01-01-01_00:00:00"')
+        lines.append('        output_interval="00-01-00_00:00:00"')
```

Comment:
@jonbob since these variables are in the monthly avg/max/min should we set output_interval='none' here?

Reply:
sure -- or else have that AM off by default?

Reply:
I think the default now is to have all the output streams off for their individual files but have them in the monthly averages.  I'd definitely like to keep this on by default.

Reply:
On but with output_interval='none'?

Reply:
yes I think that's right.

Reply:
Do you want the heat content one treated the same way? While I'm at it?

Reply:
OK, I changed the output_interval and pushed a new commit